docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,167 @@
+  is compressed. Compressing old logs with retention enabled could cause those archives to be
+  deleted immediately after compression.
+  
+  2. Archives whose log messages do not contain timestamps are not subject to retention.
+  :::
+
+### Expiry criteria
+
+- **Archive Expiry:**  
+  An archive is considered expired if its retention period has elapsed since archive's timestamp,
+  i.e. that the difference between `T` and `archive.T` has surpassed `TTL`.
+  ```text
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)